### PR TITLE
Find First and Last Position of element in sorted array in Python

### PR DESCRIPTION
**Pull request by harsha-sam**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

### Description

(Write your answer here.)

### Checklist

- [✅ ] I've read the contribution guidelines.
- [✅ ] I've checked the issue list before deciding what to submit.
- [✅ ] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests
#5685

### CODE REVIEW

File: Python/README.md [hunk 1]
```diff
@@ -117,6 +117,7 @@
 - [Hailstone Sequence](cp/Hailstone_sequence.py)
 - [Next Permutation](cp/Next_Permutation.py)
 - [Minimize Amplitude After K Removals](cp/Minimize_Amplitude_After_K_Removals.py)
+- [Find First and Last position index of an element in an array](cp/Find%20First%20and%20Last%20Position%20of%20Element%20in%20Sorted%20Array.py)
```
Comment (HarshCasper) [inline Python/README.md]: Use files-names properly. Keep underscores.

**Review by HarshCasper — CHANGES_REQUESTED** [review]

File: Python/README.md (hunk 1 above)
Comment (harsha-sam) [reply Python/README.md]: Done.

File: Python/README.md [hunk 2]
```diff
@@ -117,6 +117,7 @@
 - [Hailstone Sequence](cp/Hailstone_sequence.py)
 - [Next Permutation](cp/Next_Permutation.py)
 - [Minimize Amplitude After K Removals](cp/Minimize_Amplitude_After_K_Removals.py)
+- [Find_First_and_Last_Position_of_Element_in_Sorted_Array](cp/Find_First_and_Last_Position_of_Element_in_Sorted_Array.py)
```
Comment (ankitaggarwal23) [inline Python/README.md]: Rewrite it as,

`- [Find First and Last Position of Element in Sorted Array](cp/Find_First_and_Last_Position_of_Element_in_Sorted_Array.py)`

File: Python/cp/Find_First_and_Last_Position_of_Element_in_Sorted_Array.py [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+    we search into either of the halves depending
+    upon the result produced through the match.
+
+    - Returns the initial found index position of the target value
+      if flag is set to 0 (Default)
+    - Returns Starting Positon of the target value if flag is set to -1
+    - Returns Ending Positon of the target value if flag is set to 1
+
+    If the target is not present in the list of numbers,
+    -1 is returned irrespective of the flag value
```
Comment (ankitaggarwal23) [inline Python/cp/Find_First_and_Last_Position_of_Element_in_Sorted_Array.py]: Shift your approach inside description

File: Python/cp/Find_First_and_Last_Position_of_Element_in_Sorted_Array.py [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+
+    # Time Complexity: O(log n)
+    # Space Complexity: O(1)
```
Comment (ankitaggarwal23) [inline Python/cp/Find_First_and_Last_Position_of_Element_in_Sorted_Array.py]: Shift TC and SC at the end

File: Python/cp/Find_First_and_Last_Position_of_Element_in_Sorted_Array.py [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+
+
+# Input: nums = [5,7,7,8,8,10], target = 8
+# Output: [3,4]
+
+# Input: nums = [5,7,7,8,8,10], target = 6
+# Output: [-1,-1]
+
+# Input: nums = [], target = 0
+# Output: [-1,-1]
```
Comment (ankitaggarwal23) [inline Python/cp/Find_First_and_Last_Position_of_Element_in_Sorted_Array.py]: Use multiline comments and Use proper display message how you are reading and writing outputs

**Review by ankitaggarwal23 — CHANGES_REQUESTED** [review]

**Comment by ankitaggarwal23** [comment]
Resolve merge conflicts

File: Python/cp/Find_First_and_Last_Position_of_Element_in_Sorted_Array.py (hunk 5 above)
Comment (harsha-sam) [reply Python/cp/Find_First_and_Last_Position_of_Element_in_Sorted_Array.py]: Multiline comments are causing an anti pattern issue
Screenshot 2021-04-14 at 11 35 36 PM

**Review by ankitaggarwal23 — APPROVED** [review]

**Pull request merged**